### ISSUE TITLE
Support for \emph macro

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/tex2mn#99]
*From https://github.com/metanorma/mn-samples-tex-iso/issues/5*

`\emph{...}` command should result in *italic* text in asciidoc output.

LaTeX example:
````
\emph{emphasis}
````

Current adoc output:
````
Emphasis
````

Expected adoc output:
````
__Emphasis__
````

**Comment by paolobrasolin** [comment]
I implemented support for `\emph` in https://github.com/metanorma/tex2mn/commit/004b970bc5cdef0ab3cb48865e98107659978361

However it's not possible to reproduce its usual nesting behaviour since AsciiDoc doesn't have a way to represent that (that's why I skipped that initially in favour of `\textit` and `itshape`).

Inner levels will simply be ignored but most use cases should be covered. This is made clear in the manual.

cc @manuel489
